docs/shared/ungrib.yaml

```diff
@@ -0,0 +1,13 @@
+ungrib:
+  execution:
+    mpicmd: srun
+    batchargs:
+      nodes: 1
+      walltime: 00:05:00
```

Comment:
IIRC `00:01:00` might be enough for `ungrib`. As above, this could be a useful hint to users about what `ungrib`'s actual requirements are.